features/cxf/src/main/feature/feature.xml

```diff
@@ -365,66 +434,87 @@
         </bundle>
-        <bundle start-level="35">
-            mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/${cxf.jackson.version}
+        <!-- <bundle start-level="35">mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-base/${cxf.jackson.version}</bundle> -->
+        <!-- <bundle start-level="35">mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/${cxf.jackson.version}</bundle> -->
+        <feature prerequisite="true">wrap</feature>
+        <bundle>
+            wrap:mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-base/${cxf.jackson.version}$overwrite=merge&amp;Import-Package=javax.ws.rs*;version="[2.0,3)",com.fasterxml.jackson*;version="[2.8,3)"
         </bundle>
-        <bundle start-level="35">
-            mvn:com.fasterxml.jackson.module/jackson-module-jaxb-annotations/${cxf.jackson.version}
+        <bundle>
+            wrap:mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/${cxf.jackson.version}$overwrite=merge&amp;Import-Package=javax.ws.rs*;version="[2.0,3)",com.fasterxml.jackson.module.jaxb;resolution:=optional;version="[2.8,3)",com.fasterxml.jackson*;version="[2.8,3)"
+        </bundle>
+    </feature>
+    <feature name="cxf-jsr-json" version="${cxf.version}">
+        <bundle start-level="35">mvn:javax.json/javax.json-api/1.0</bundle>
+        <bundle start-level="35">mvn:org.apache.johnzon/johnzon-core/1.0.0</bundle>
+    </feature>
+    <feature name="cxf-tracing-brave" version="${cxf.version}">
+        <feature version="${cxf.version}">cxf-core</feature>
+        <bundle start-level="10" dependency="true">mvn:com.google.code.findbugs/jsr305/3.0.2
+        </bundle>
+        <bundle start-level="35">mvn:io.zipkin.java/zipkin/1.31.1</bundle>
+        <bundle start-level="35">mvn:io.zipkin.reporter/zipkin-reporter/1.0.2</bundle>
+        <bundle start-level="35">mvn:io.zipkin.brave/brave/4.6.0</bundle>
+        <bundle start-level="35">mvn:io.zipkin.brave/brave-instrumentation-http/4.6.0</bundle>
+        <bundle start-level="40">mvn:org.apache.cxf/cxf-integration-tracing-brave/${cxf.version}
+        </bundle>
+    </feature>
+    <feature name="cxf-tracing-opentracing" version="${cxf.version}">
+        <feature version="${cxf.version}">cxf-core</feature>
+        <bundle start-level="35">wrap:mvn:io.opentracing/opentracing-api/0.30.0</bundle>
+        <bundle start-level="35">wrap:mvn:io.opentracing/opentracing-util/0.30.0</bundle>
+        <bundle start-level="40">
+            mvn:org.apache.cxf/cxf-integration-tracing-opentracing/${cxf.version}
         </bundle>
     </feature>
     <feature name="cxf-rs-description-swagger2" version="${cxf.version}">
         <feature version="${cxf.version}">cxf-jaxrs</feature>
         <feature version="${cxf.version}">cxf-jackson</feature>
+        <bundle start-level="35">
+            mvn:org.apache.cxf/cxf-rt-rs-service-description-swagger-ui/${cxf.version}
+        </bundle>
```

Comment:
Swagger integration?  Will it replace what we see when we hit /services?

Reply:
Again, most likely just in use by other CXF libraries.